sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelperTest.java

```diff
@@ -467,4 +469,35 @@ public void throwJSONExceptionWhenGetMinidumpSubfolderWithDeviceInfo() throws ja
         ErrorLogHelper.getNewMinidumpSubfolderWithContextData(mockContext);
         verify(mockFile).delete();
     }
+
+    @Test
+    public void parseLogFolderUuid() {
+        String originalFolderName = "a80da2ae-8c85-43b0-a25b-d52319fb6d56";
+        File logFolder = new File("lib/files/error/minidump/new/" + originalFolderName);
+        File spy = spy(logFolder);
```

Comment:
Do not use spy and mock for files.
Use
```
    @Rule
    public TemporaryFolder mTemporaryFolder = new TemporaryFolder();
```

See how it was implemented in distribute tests as example